hack/lib/update-codegen.sh

```diff
@@ -98,33 +102,40 @@ require (
     k8s.io/api ${KUBE_MODULE_VERSION}
     k8s.io/apimachinery ${KUBE_MODULE_VERSION}
     k8s.io/client-go ${KUBE_MODULE_VERSION}
-    k8s.io/apimachinery ${KUBE_MODULE_VERSION}
+    ${BASE_PKG}/generated/${KUBE_MINOR_VERSION}/apis v0.0.0
 )
 
 replace ${BASE_PKG}/generated/${KUBE_MINOR_VERSION}/apis => ../apis
 EOF
 
+# Generate a go.sum without changing the go.mod by running go mod download.
+echo "running go mod download in ${OUTPUT_DIR}/client/go.mod to generate a go.sum file..."
+(cd "${OUTPUT_DIR}/client" && go mod download all 2>&1 | sed "s|^|go-mod-download > |")
+
+# Note that you can change this value to 10 to debug the Kubernetes code generator shell scripts used below.
+debug_level=1
```

Comment:
```suggestion
debug_level="${CODEGEN_LOG_LEVEL:-1}"
```

And have CI `export CODEGEN_LOG_LEVEL=10` so it is always dumping full logs?